src/app.provider.ts

```diff
@@ -24,11 +24,15 @@ function configureSwagger(app: INestApplication) {
     .build();
 
   const document = SwaggerModule.createDocument(app, config);
-  SwaggerModule.setup('', app, document, {
+
+  const options = {
     customfavIcon: '/favicon.png',
     customSiteTitle: 'Safe Client Gateway',
     customCss: `.topbar-wrapper img { content:url(\'logo.svg\'); }`,
-  });
+  };
+
+  SwaggerModule.setup('/index.html', app, document, options);
+  SwaggerModule.setup('', app, document, options);
```

Comment:
I've checked the Nest source code for the `setup` function and it calls both `serveDocuments` and `serveStatic`, which in the end call `httpAdapter.get`. This `httpAdapter` (`node_modules/@nestjs/core/adapters/http-adapter.js`) seems to be a wrapper class that calls Express functions (`get` in this case). So to summarize, I think this is just setting up two routes in the underlying Express router. I guess the only performance impact this could bring is on the app startup (and it should be trivial).

Having said that, I agree with you maybe it's better to work on a simple redirection so we don't depend on a library function which could change over time. Are you thinking of setting up an internal redirection in the service? Nginx?

Reply:
Agreed, closing this PR then in order to create another to apply the redirection on the reverse proxy 👍